src/pages/signin.tsx

```diff
@@ -0,0 +1,78 @@
+import { Link } from 'gatsby';
+import * as React from 'react';
+
+import styled from '@styled-components';
+import { Layout, Seo } from '@components/shared';
+import CtaButton from '@components/index-page/cta-button';
+import { useSiteMetadata } from '@hooks';
+import Form, {
+  FormLabel,
+  FormInput,
+  LinkWrapper,
+  ButtonWrapper,
+} from '@components/shared/form';
+import { useState } from 'react';
+
+const Wrapper = styled.section`
+  background-color: ${({ theme }) => theme.colors.section};
+
+  padding: ${({ theme }) => theme.boxes.padding.section.smallTop};
+
+  @media screen and(max-width: ${({ theme }) => theme.sizes.width.medium}) {
+    flex-direction: column;
+  }
+
+  @media screen and(max-width: ${({ theme }) => theme.sizes.width.small}) {
+    padding: ${({ theme }) => theme.boxes.padding.section.small};
+  }
+`;
+
+const SignInPage: React.FC = () => {
+  const siteMetadata = useSiteMetadata();
+
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  return (
+    <Layout>
+      <Seo
+        title={`Sign In`}
+        description={`Sign In page for ${siteMetadata.title} website`}
+        urlSlug="signin/"
+      />
+      <Wrapper>
+        <Form heading="Sign In To Project Unicorn">
```

Comment:
[1] Hard-coded "Project Unicorn" string can be replaced by `siteMetadata.title`

```suggestion
        <Form heading={`Sign In To ${siteMetadata.title}`}>
```